INSTALL.md

```diff
@@ -189,9 +527,9 @@ Configuration file example of ICON RPC Server.
 
 | Field | Type | Description |
 |:------|------|:------------|
-| log | dict | Refer to the description of iconservice_config.json |
+| log | dict | Refer to the [Configuration Files > ICON Service](#icon-service) section |
 | channel | string | channel name to interact with loopchain |
-| amqpKey | string | used for a part of a queue name of RabbitMQ |
+| amqpKey | string | used for part of a queue name of RabbitMQ |
```

Comment:
can't get the meaning. any string value possibly duplicate is acceptable for amqpKey ?

Reply:
ICON Service component creates a named queue in rabbitmq.
And the name of the queue is made in the following rule: "IconScore.{channel}.{amqpKey}"
ex) channel = "icon_dex", amqpKey = "peer0", queue name = "IconScore.icon_dex.peer0"

If ICON RPC Server publishes a message to the named queue, ICON Service component consumes and processes the message. And then it returns the result to the ICON RPC Server.

Reply:
In the example configuration file, the value seems the loopchain port number, so I was curious if the amqpKey must match the actual port number used or not. 
After looking further into this, it looks like the "channel" and "amqpKey" must match the values of  "LOOPCHAIN_DEFAULT_CHANNEL" and "AMQP_KEY" in the loopchain configuration file. If so, It would be much nicer to mention it. 

In the "Example: Execute 1 RadioStation and 4 Peers on localhost", what is a recommended configuration ? a) each peer has different channel and amqpKey, b) each has same channel and different amqpKey, c) each has same channel and amqpKey (maybe not)